yarn.lock

```diff
@@ -1592,9 +1592,9 @@ create-hmac@^1.1.0, create-hmac@^1.1.2, create-hmac@^1.1.4:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
-create-react-context@^0.2.1:
-  version "0.2.3"
-  resolved "https://registry.yarnpkg.com/create-react-context/-/create-react-context-0.2.3.tgz#9ec140a6914a22ef04b8b09b7771de89567cb6f3"
+create-react-context@0.2.2, create-react-context@^0.2.1:
```

Comment:
this change seems to be unnecessary, let me try this out..

Reply:
this was generated by yarn itself, only change I made was in `package.json`